internal/pkg/workspace/workspace.go

```diff
@@ -468,11 +468,6 @@ func (ws *Workspace) ListDockerfiles() ([]string, error) {
 			}
 		}
 	}
-	if len(directories) == 0 {
```

Comment:
nice i like this change of returning just an empty list instead of an error if no dockerfiles are found